Environment/Environment.tex

```diff
@@ -764,6 +764,58 @@ \subsection{The object explorer}
 	\label{fig:explorePluggableListMorph}
 \end{figure}
 
+
+\subsection{The instance browser}
+
+Sometimes, you might want to explore what a specific object can do.
+In the class browser, you can look through the methods to learn what they should do, but you can not execute them and see what they actually do.
+In the inspector or explorer, you can see the state of an object and execute methods, but you can not directly see the code corresponding to the methods.
+In case you want both, reading code and evaluating code for a single object, you can use the \emph{instance browser} tool.
+\index{instance browser}
+The instance browser allows you to read the code of all methods that an object can execute, and evaluate any code snippets in the context of this object.
+
+You can open an instance browser from the explorer.
+\dothis{In the explorer above, select the root entry for the \ct{PluggableListMorph}, open the yellow button menu, and choose \menu{browse protocol (p)}.}

```

Comment:
Surround with newlines (should probably not look like this: ![image](https://github.com/hpi-swa-lab/SqueakByExample-english/assets/38782922/e140af4c-38d8-4420-b3b8-5450b9f1d9ee))